src/Avalonia.Controls.ColorPicker/Themes/Simple/ColorPicker.xaml

```diff
@@ -50,7 +52,8 @@
             </Style>
           </DropDownButton.Styles>
           <DropDownButton.Flyout>
-            <Flyout Placement="Top">
+            <Flyout FlyoutPresenterClasses="nopadding"
```

Comment:
Are you sure this is defined for simple theme? It was likely not set on purpose.